scripts/o3de/o3de/compatibility.py

```diff
@@ -326,3 +369,173 @@ def has_compatible_version(name_and_version_specifier_list:list, object_name:str
             pass
 
     return False 
+
+class GemRequirement(namedtuple("GemRequirement", ["name", "specifier"])):  # noqa
```

Comment:
what is `#noqa` ?

Reply:
oh... that's a comment from resolvelib's code I used initially - will remove